client/src/components/nav/nav.styles.js

```diff
@@ -1,49 +1,107 @@
 import styled from 'styled-components';
 
-export const NavSection = styled.ul`
-  display: flex;
-  
-  align-items: center;
+
+export const NavContainer = styled.div`
+  z-index: 12;
   width: 100%;
-  padding: 0;
-  margin: 0;
-  height: 60px;
- 
-  > li {
-    width: auto;
-    list-style: none;
-    padding: 10px;
-    font-size: 14px;
-    text-align: center;
-    text-decoration: none;
-    color: #333333;
-    font-weight: 700;
-    > span {
-      cursor: pointer;
-    }
+  font-family: "Montserrat", sans-serif;
+  @media only screen and (max-width: 480px) { 
+    height: auto;
+    width: 100%;
   }
+`;
 
+// Start of nav top
+export const NavTop = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
```

Comment:
Just a tiny detail for following the mock-up design, but to just give a space between the top and bottom nav and between top page and top nav, I suggest adding a line, 'padding: 15px 0px'.